packages/core/src/App/Containers/AccountTypesModal/account-types-modal.jsx

```diff
@@ -0,0 +1,313 @@
+import { Icon, Modal, Tabs } from '@deriv/components';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { localize, Localize } from '@deriv/translations';
+import AccountCard from './account-card.jsx';
+
+import 'Sass/app/modules/account-types.scss';
+
+const Box = ({ title, description, footer_text, icons, cards }) => {
+    return (
+        <div className='account-types__box'>
+            <div className='account-types__box-left'>
+                <h2 className='account-types__box-title'>{title}</h2>
+                <p className='account-types__box-description'>{description}</p>
+                {footer_text && <p className='account-types__box-footer'>{footer_text}</p>}
+                <div className='account-types__box-icons'>
+                    {icons.map((icon, index) => {
+                        return <Icon className='account-types__box-icon' icon={icon} key={index} />;
+                    })}
+                </div>
+            </div>
+            {cards}
+        </div>
+    );
+};
+
+Box.propTypes = {
+    title: PropTypes.string,
+    description: PropTypes.string,
+    footer_text: PropTypes.string,
+    icons: PropTypes.arrayOf(PropTypes.string),
+    cards: PropTypes.array,
+};
+
+const FinancialBox = ({ is_demo = false }) => {
+    return (
+        <Box
+            title={localize('Financial account ({{type}})', { type: is_demo ? localize('Demo') : localize('Real') })}
+            description={localize(
+                'Financial accounts offer you to trade financial assets such as Forex, commodities and indices.'
+            )}
+            footer_text={localize('Over 50 tradable financial assets')}
+            icons={['IcUnderlyingFRXNZDJPY', 'IcUnderlyingOTC_NDX', 'IcUnderlyingFRXXAUUSD', 'IcUnderlyingFRXBROUSD']}
+            cards={[
+                <AccountCard
+                    key={0}
+                    title={localize('Trade on Deriv')}
+                    subtitle={localize('Option trading account')}
+                    button_text={is_demo ? localize('Add demo account') : localize('Add real account')}
```

Comment:
What happens if the client is logged in as virtual? Should we change the buttons?

Reply:
@reza We don't know yet. since this is just UI, let keep it for now and change it when implementing logic